src/app/forms/templates/test-records/master.template.ts

```diff
@@ -456,13 +457,33 @@ export const masterTpl: Record<VehicleTypes, Partial<Record<keyof typeof TEST_TY
   },
   lgv: {
     default: {
-      required: RequiredSpecialistSection,
+      required: SpecialistRequiredSectionLgvCar,
       vehicle: VehicleSectionDefaultPsvHgvLight,
       test: TestSection,
       defects: defectsHiddenSection,
       visit: VisitSection,
       notes: NotesSection,
       reasonForCreation: reasonForCreationSection
+    },
+    testTypesSpecialistGroup1: {
+      required: SpecialistRequiredSectionLgvCar,
+      vehicle: VehicleSectionDefaultPsvHgvLight,
+      test: SpecialistTestSectionGroup1,
+      defects: DefectsTpl,
+      customDefects: CustomDefectsHiddenSection,
+      visit: VisitSection,
+      notes: NotesSection,
+      reasonForCreation: reasonForCreationSection
+    },
+    testTypesSpecialistGroup5: {
```

Comment:
Is this supposed to be testTypesSpecialistGroup**2**?

Reply:
The testTypeIDs for this group map to testTypesSpecialistGroup5, as they're Voluntary IVA tests